docs/browser/events.md

```diff
@@ -0,0 +1,158 @@
+| fetchStart | long | || Recommended |
+| unloadEventStart | long | || Recommended |
+| unloadEventEnd | long | || Recommended |
+| domInteractive | long | || Recommended |
+| domContentLoadedEventStart | long | || Recommended |
+| domContentLoadedEventEnd | long | || Recommended |
+| domComplete | long | || Recommended |
+| loadEventStart | long | || Recommended |
+| loadEventEnd | long | || Recommended |
+| firstPaint | long | || Recommended |
+| firstContentfulPaint | long | || Recommended |
+
+## ResourceTiming
+
+The event name MUST be `resource_timing`.
+
+This event describes the timing metrics as provided by `PerformanceResourceTiming`
+Performance API.  These metrics are related to fetching a resource, such as
+XMLHttpRequest, Fetch, sendBeacon APIs, SVG, image or script.
+
+The following table describes the payload fields that MUST be used to describe the details of event.
+
+| Body Field  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+|name | string | URL of the requested resource || Recommended |
+|fetchStart | long | || Recommended |
+|domainLookupStart | long | || Recommended |
+|domainLookupEnd | long | || Recommended |
+|connectStart | long | || Recommended |
+|secureConnectionStart | long | || Recommended |
+|connectEnd | long | || Recommended |
+|requestStart | long | || Recommended |
+|responseStart | long | || Recommended |
+|responseEnd | long | || Recommended |
+
+## Exception
+
+The event name MUST be `browser.exception`.
+
+This event describes an error or exception that occurs in a browser application.
+
+The following table describes the payload fields that MUST be used to describe the details of event.
+
+| Body Field  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `exception.file_name` | string | Name of the file that generated the error | `foo.js` | Recommended |
+| `exception.line_number` | int | Line number where the error occurred |  | Recommended |
+| `exception.column_number` | int | Column number where the error occurred |  | Recommended |
+| `exception.message` | string | The exception message. | `Division by zero`; `Can't convert 'int' object to str implicitly` | Recommended |
+| `exception.stacktrace` | string | A stacktrace as a string in the natural representation for the language runtime. The representation is to be determined and documented by each language SIG. | `Exception in thread "main" java.lang.RuntimeException: Test exception\n at com.example.GenerateTrace.methodB(GenerateTrace.java:13)\n at com.example.GenerateTrace.methodA(GenerateTrace.java:9)\n at com.example.GenerateTrace.main(GenerateTrace.java:5)` | Recommended |
+| `exception.type` | string | The type of the exception (its fully-qualified class name, if applicable). The dynamic type of the exception should be preferred over the static type in languages that support it. | `java.net.ConnectException`; `OSError` | Recommended |
```

Comment:
**Suggestion (could be done later):** Some frameworks ([but most notably React](https://react.dev/reference/react/Component#componentdidcatch-parameters)) expose an alternative stack trace format that is quite handy when debugging called a `componentStack`.

A `componentStack` cuts through the framework stack frames and instead shows the component nesting directly.

<img width="294" alt="image" src="https://github.com/open-telemetry/semantic-conventions/assets/596443/0decc356-3eb0-478f-923d-1423037b3a54">

Reply:
> **Suggestion (could be done later):** Some frameworks ([but most notably React](https://react.dev/reference/react/Component#componentdidcatch-parameters)) expose an alternative stack trace format that is quite handy when debugging called a `componentStack`.
> 
> A `componentStack` cuts through the framework stack frames and instead shows the component nesting directly.
> 
> It would imho make sense to also specific an attribute for this use case.

@bripkens I have moved this event out into a separate PR - https://github.com/open-telemetry/semantic-conventions/pull/970 - we decided in the Event SIG to have a common Event for exceptions.

The PR still uses attributes for the exception data, but I guess using Event body for capturing exception data would allow for adding a `context(s)` field for attaching arbitrary additional data such as the componentStack. This woud be a good discussion to have in the Event SIG.